src/main/webapp/components/Navbar.vue

```diff
@@ -1,46 +1,65 @@
 <template>
   <nav class="navy-blue sticky nav-bar">
     <a href="index.html" class="bold" id="title">starfish</a>
-    <img src="../assets/starfish.png" id="logo">
+    <img src="/assets/starfish.png" id="logo">
     <div class="action-items" v-if="!signedIn">
-      <a href="#" class="vert-center" @click.stop="signIn" id="signin-link">Sign Up/Login</a>
+      <v-btn color="primary"
+             dark
+             @click.stop="signIn"
+             id="signin-link">
+        Sign Up/Login
+      </v-btn>
     </div>
     <div class="action-items" v-if="signedIn">
-      <a href="#" class="vert-center">PostANote</a>
-      <div class="dropdown">
-        <button class="dropdown-btn" @click.stop="toggleDropdown">
-          <img id="user-profile" src="../assets/user.svg">
```

Comment:
Should this path be `/assets/user.svg` instead? (Asking because the `..` was removed from the path to `assets` above)

Reply:
I think all of this code should be deleted, so we actually don't need to worry about it 😄